site/layouts/shortcodes/k8s-api.html

```diff
@@ -1,2 +1,7 @@
-{{- $version := cond (eq (getenv "HUGO_ENV") "production") $.Page.Site.Params.supported_k8s $.Page.Site.Params.dev_supported_k8s }}
-{{- $prefix := replace $version "." "-" }}https://v{{- $prefix }}.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v{{- $version }}/{{- .Get "href" }}
\ No newline at end of file
+<!-- $version must be assigned outside of if statement for variable scope purposes.
```

Comment:
There's somethign in this that creating a linebreak, you can see it here:

![image](https://user-images.githubusercontent.com/298370/215218595-70f1ed6f-602c-4655-a29f-0ba5f7bd1ef8.png)

http://localhost:1313/docs/reference/fleet/

It could be HTML docs at the top, or it might be that `{{ end }}` should be `{{- end }}`. Whitespace and line breaks can be tricky in Hugo sometimes. Could also be linebreak we didn't have previously on the end of the shortcode.